### ISSUE TITLE
Test suite failure with latest hlint

### ISSUE THREAD

**Issue by snoyberg** [ekmett/lens#634]
I see a number of failures with the latest release of hlint, e.g.:

```
lens-4.13.1: test (suite: hlint)
               
src/Control/Lens/At.hs:6:1: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE DefaultSignatures #-}
Why not remove it.

src/Control/Lens/Each.hs:6:1: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE DefaultSignatures #-}
Why not remove it.

src/Control/Lens/Empty.hs:3:1: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE DefaultSignatures #-}
Why not remove it.

src/Control/Lens/Fold.hs:932:15: Warning: Use fmap
Found:
  liftM
Why not:
  fmap

src/Control/Lens/Fold.hs:976:17: Warning: Use fmap
Found:
  liftM
Why not:
  fmap

src/Control/Lens/Fold.hs:2185:16: Warning: Use fmap
Found:
  liftM
Why not:
  fmap

src/Control/Lens/Indexed.hs:6:1: Warning: Unused LANGUAGE pragma
Found:
  {-# LANGUAGE DefaultSignatures #-}
Why not remove it.
```

**Comment by snoyberg** [comment]
Can you ping this issue when this fix is released to Hackage?

**Comment by glguy** [comment]
@snoyberg lens-4.13.1 on Hackage has been updated to exclude the new version of hlint that causes the test to fail.

**Comment by glguy** [comment]
@snoyberg Fixed in 4.13.2
